scripts/add_release_annotation.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# TODO(tflannag): Add ability to override yq binary as either a parameter
+# or an environment varaibles like upstream OLM does.
+# For now, you can get the yq binary by running `go install ./vendor/github.com/mikefarah/yq/v3`
+# from the root repository and moving that binary into $PATH.
+chartdir=$1
+
+for f in $chartdir/*.yaml; do
```

Comment:
I know it's not really the topic of this PR to discuss design, but I don't really see a reason for this to be a script instead of a branch in our Helm templates; i.e. when `.Values.imagestream=true`: https://github.com/openshift/operator-framework-olm/blob/2df1ac17508e41b3191b04bc3fb00a30cbad681f/staging/operator-lifecycle-manager/deploy/chart/templates/0000_50_olm_99-operatorstatus.yaml#L1

Reply:
Agreed - I may take some time in the next couple of weeks to tackle updating the layout of the charts + generate script(s).